environments/gym_adapter.py

```diff
@@ -100,7 +102,7 @@ async def impl(self, environment_session):
                 if isinstance(gym_env.action_space, gym.spaces.Box):
                     action_value = np.clip(action_value, gym_env.action_space.low, gym_env.action_space.high)
 
-                gym_observation, reward, done, _info = gym_env.step(action_value)
+                gym_observation, reward, terminated, truncated, _info = gym_env.step(action_value)
```

Comment:
@will I think we should do the same for `pettingzoo_adapter` as well. Maybe not this PR but the one where we want to migrate to `gymnasium`

Reply:
I took a look and I'm not sure the changes can be applied to pettingzoo environments because we do not use the `gym.make()` method.

Reply:
> @will I think we should do the same for `pettingzoo_adapter` as well. Maybe not this PR but the one where we want to migrate to `gymnasium`

Wouldn't it depend on what kind of animals you have at the petting zoo? Why would like sheep and stuff need to go to a gym.